features/other_a/languages.feature

```diff
@@ -1,5 +1,32 @@
 @works
 Feature: Languages
+
+  Scenario: Add a language and check abuse and support available
```

Comment:
Looking at     https://github.com/otwcode/otwarchive/blob/d1858d91cfacd9b6c23d3079538bd61db2859e44/features/admins/admin_languages.feature

And I go to the languages page
    And I follow "Edit"

So I do need a test for creating a language. I am willing to drop the test on seeing if it is in abuse if you want ?